deployments/mainnet/usdc/roots.json

```diff
@@ -6,5 +6,7 @@
     "bulker": "0xa397a8C2086C554B531c02E29f3291c9704B00c7",
     "fxRoot": "0xfe5e5D361b2ad62c541bAb87C45a0B9B018389a2",
     "arbitrumInbox": "0x4Dbd4fc535Ac27206064B68FfCf827b0A60BAB3f",
-    "arbitrumL1GatewayRouter": "0x72Ce9c846789fdB6fC1f34aC4AD25Dd9ef7031ef"
+    "arbitrumL1GatewayRouter": "0x72Ce9c846789fdB6fC1f34aC4AD25Dd9ef7031ef", 
+    "mainnetCCTPTokenMessenger": "0xbd3fa81b58ba92a82136038b25adec7066af3155", 
+    "mainnetCCTPMessageTransmitter": "0x0a992d191deec32afe36203ad87d7d289a738f81"
```

Comment:
nit: don't think we need to prepend with `mainnet` to the names here since the this is the roots for mainnet.

Reply:
Removed 👍